app/assets/javascripts/jquery_overrides.js

```diff
@@ -1,6 +1,10 @@
 jQuery.evalLog = function (text) {
   try {
-    jQuery.globalEval(text.slice('throw "error";'.length));
+    if (text.match('^throw "error";')) {
```

Comment:
@abellotti, @himdel :
The blind slicing it there on purpose. If it was optional it would be only a matter of time until someone somwhere forgets to add the prologue. And everything will work, noone will notice.
Now there's pretty good chance that if someone forgets to add the prologue, his/her changes will not work so he/she will notice and fix it.

Please, do not change this,